common/api/imodeljs-common.api.md

```diff
@@ -2326,6 +2336,26 @@ export interface ElementProps extends EntityProps {
     userLabel?: string;
 }
 
+// @public
+export interface EmphasizeElementsProps {
+    // (undocumented)
+    alwaysDrawn?: Id64Array;

```

Comment:
`@public` properties should be documented. For "props" types like this it often suffices to put, for example, `@see [EmphasizeElements.unanimatedAppearance]($frontend)`.

Reply:
Sounds good.